### PR TITLE
chore(test): Add inventory tests on GetHelmRelease

### PR DESCRIPTION
**Pull request by yitsushi**
Extra tests for #1534 

(Merged #1534 to make sure I'm not blocking any UI related works while I'm trying to figure out how to test this properly)

### CODE REVIEW

File: core/server/helm_release_test.go [hunk 1]
```diff
@@ -82,6 +90,199 @@ func TestGetHelmRelease(t *testing.T) {
...
+
+	g.Expect(k.Status().Patch(ctx, &helmRelease, client.Apply, opt...)).To(Succeed())
+
+	// Create helm storage.
+	storage := types.HelmReleaseStorage{
+		Name:     "",
+		Manifest: helmReleaseInventoryObjects(),
+	}
+
+	storageData, _ := json.Marshal(storage)
```
Comment (Callisto13) [inline core/server/helm_release_test.go:231]: missing err check?

File: core/server/helm_release_test.go [hunk 2]
```diff
@@ -82,6 +90,199 @@ func TestGetHelmRelease(t *testing.T) {
...
+
+	var compressed bytes.Buffer
+
+	compressor := gzip.NewWriter(&compressed)
+	_, _ = compressor.Write(storageData)
+	compressor.Close()
```
Comment (Callisto13) [inline core/server/helm_release_test.go:237]: any values we need to check here?

File: core/server/helm_release_test.go (hunk 1 above)
Comment (yitsushi) [reply core/server/helm_release_test.go:231]: not missing... ignored on purpose.

File: core/server/helm_release_test.go (hunk 2 above)
Comment (yitsushi) [reply core/server/helm_release_test.go:237]: Well, if we don't test compressed data, we can't be sure it's working.

Comment (Callisto13) [reply core/server/helm_release_test.go:237]: no i mean, does `Write` return anything we might care about? like an error

Comment (yitsushi) [reply core/server/helm_release_test.go:237]: it returns with the number of bytes written and an error if it failed, but it shouldn't fail, it's a write and we are not using filesystem io, buffer writing will not fail, if it does somehow, the test will fail anyway.

Comment (yitsushi) [reply core/server/helm_release_test.go:237]: The reason I put the `_, _ =` there because linter cried.

**Review by jpellizzari — APPROVED** [review]
Nice work!

**Pull request merged**